openqabot/incsyncres.py

```diff
@@ -68,7 +68,11 @@ def __call__(self) -> int:
                 results.append(r)
 
         for r in results:
-            logger.info("Posting job results to dashboard: %s" % pformat(r))
+            logger.info(
+                "Posting results of incident job %s with status %"
+                % (r["job_id"], r["status"])
+            )
+            logger.debug("Full post data: %s" % pformat(r))
```

Comment:
Maybe a small change to make it more similar to aggregates messages?

```suggestion
            logger.debug("Full incident job result to dashboard: %s" % pformat(r))
```

Reply:
this info is in logger.info above ... so it will be only duplicate